### DISCUSSION TITLE
Fetch HTML meta tags while capture page snapshot

### DISCUSSION THREAD

**Discussion by trinityzhub** [coddingtonbear/obsidian-web#216] [Ideas]
I am newbie in obsidian, Is it possible get metadata in page properties

```
---
page-title: {{json page.title}}
url: {{page.url}}
url-aliases:<<<<<<<<<<<<  {{page.url}}.trimToBaseUrl()+"/*" >>>>>>>>>>>>>>>
date: "{{date}}"
type:"article"
meta-keywords: <<<<<<<<<< << array of tags >>>>>>>>>>>>>>>
meta-description:  <<<<<<<<<<<<< < json page.description  >>>>>>>>>>>>>>
---
{{#if page.selectedText}}

{{quote page.selectedText}}

---

{{/if}}{{page.content}}


```

**Comment by coddingtonbear** [comment]
I'm afraid that isn't possible right now, but I'm interested in hearing more about your usecase and why that might be helpful for you!

**Comment by Dnarts** [comment]
This has the potential to flood your vault with useless tags; I would be wary of these...

**Comment by trinityzhub** [comment]
Regarding `url-aliases`

when i am visiting domain1.com, It will help me to list all notes related to that domain in a dialog box. (Feature of "Automatically Display Matches")
after2 years. **i may forgot my visit on an article of this website "domain1.com". But obsidian web will recollect.**



Regarding `meta-keywords`:
It might help for Data view queries and search.

**It might cool and fun** for who needs. **But NOT NECCESSORY.**
Please ignore this thread
